01-lets_get_started/01.rb

```diff
@@ -26,4 +26,4 @@ def generate
 end
```

Comment:
The new line should be below this, not above. Use `rubocop` to check for style guide violations.